gradio/components/chatbot.py

```diff
@@ -238,7 +238,7 @@ def __init__(
         show_copy_all_button=False,
         allow_file_downloads=True,
         group_consecutive_messages: bool = True,
-        allow_tags: list[str] | None = None,
+        allow_tags: list[str] | bool | None = None,
```

Comment:
Given the docstring, should we change to this:
```suggestion
        allow_tags: list[str] | bool = False,
```

Reply:
Wouldn't this be a breaking change?

Reply:
I guess you are right if a user has _explicitly_ set `allow_tags` to `None`, but I doubt that would be the case. Even so what we could do is change the signature to `allow_tags: list[str] | bool = False` for simplicity and then in the `__init__`, treat any falsey value as `False`, so `[]`, `False`, and `None` are all treated as `False`